src/lib/__tests__/graphqlErrorHandler.test.ts

```diff
@@ -58,6 +59,22 @@ describe("graphqlErrorHandler", () => {
   })
 
   describe(formattedGraphQLError, () => {
+    describe("stack traces", () => {
+      it("are not present in production", () => {
+        config.PRODUCTION_ENV = true
+        expect(
+          Object.keys(formattedGraphQLError(new GraphQLError("something")))
+        ).not.toContain("stack")
+        config.PRODUCTION_ENV = false
```

Comment:
FYI This is not a safe way to reset global state. If the above assertion fails it will throw an error and this code will never be executed. You should either use a `try { … } finally { … }` block or use a `after(All|Each)` block.